exercises/practice/acronym/test-words.8th

```diff
@@ -32,7 +32,7 @@ true var, run-test
 ;
 
 : run-test? \ -- T
-   run-test @ if true else "RUN_ALL_TESTS" getenv n:>bool then
+   run-test @ if true else "RUN_ALL_TESTS" getenv >n n:>bool then
```

Comment:
This breaks what is documented to work.

Reply:
I've reverted this change, sorry I didn't read clearly.